collections/individual/linkedresources.php

```diff
@@ -117,34 +117,36 @@
 		else echo '<div style="margin:15px 0px">'.(isset($LANG['OCCURRENCENOTLINKED'])?$LANG['OCCURRENCENOTLINKED']:'Occurrence is not linked to any datasets').'</div>';
 		if($SYMB_UID){
 			?>
-			<fieldset id="dataset-block" style="display:none">
-				<legend><?php echo (isset($LANG['CREATENEWREL'])?$LANG['CREATENEWREL']:'Create New Dataset Relationship'); ?></legend>
-				<form action="../datasets/datasetHandler.php" method="post" onsubmit="return verifyDatasetForm(this);">
-					<div style="margin:3px">
-						<select name="targetdatasetid">
-							<option value=""><?php echo (isset($LANG['SELECTEXISTING'])?$LANG['SELECTEXISTING']:'Select an Existing Dataset'); ?></option>
-							<option value="">----------------------------------</option>
-							<?php
-							foreach($datasetArr as $dsid => $dsArr){
-								if(!array_key_exists('linked',$dsArr)){
-									echo '<option value="'.$dsid.'">'.$dsArr['name'].'</option>';
+			<div class="dataset-block" id="dataset-block" style="display: none;">
```

Comment:
Just added a `div` here and gave it some of the attributes of the (now child) `fieldset` element.